docs/essentials/test-summary.md

```diff
@@ -1,110 +1,129 @@
+| JavaScript | Jest         | [jest-junit&nbsp;↗](https://www.npmjs.com/package/jest-junit){target="_blank"}                                        | [Exporting XML reports with Jest]()
+| Ruby       | RSpec        | [rspec_junit_formatter&nbsp;↗](https://github.com/victorolinasc/junit-formatter){target="_blank"}                     | [Exporting XML reports with RSpec]()
+| Ruby       | Cucumber     | [built-in&nbsp;↗](https://relishapp.com/cucumber/cucumber/docs/formatters/junit-output-formatter){target="_blank"}    | [Exporting XML reports with Cucumber]()
+| Elixir     | ExUnit       | [junit-formatter&nbsp;↗](https://github.com/victorolinasc/junit-formatter){target="_blank"}                           | [Exporting XML reports from ExUnit]()
+| Go         | GoTestSum    | [built-in&nbsp;↗](https://github.com/gotestyourself/gotestsum#junit-xml-output){target="_blank"}                      | [Exporting XML reports with GoTestSum]()
+| PHP        | PHPUnit      | [built-in&nbsp;↗](https://phpunit.readthedocs.io/en/9.5/textui.html?highlight=junit){target="_blank"}                 | [Exporting XML reports with PHPUnit]()
+| Python     | PyTest       | [built-in&nbsp;↗](https://docs.pytest.org/en/6.2.x/usage.html#creating-junitxml-format-files){target="_blank"}        | [Exporting XML reports with PyTest]()
+| Bash       | Bats         | [built-in&nbsp;↗](https://bats-core.readthedocs.io/en/latest/usage.html){target="_blank"}                             | &mdash;
+| Rust       | Cargo Test   | [junit-report&nbsp;↗](https://crates.io/crates/junit-report){target="_blank"}                                         | &mdash;
+| Java       | Maven        | [maven-surefire&nbsp;↗](https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit.html){target="_blank"} | &mdash;
 
-After this step is complete, you need to compile and publish test results to artifact store. Usualy it's being done in `epilogue` block:
+If your test runner is not the above list, you can still collect Tests. Find a
+test formatter for your test runner that can export JUnit XML results.
 
-```yaml
-epilogue:
-  always:
-    commands:
-      - test-results publish PATH_TO_YOUR_JUNIT_FILE
+### Step 2 &mdash; Publish XML results from your jobs
+
+Given that your test suite is exporting an XML test result named `report.xml`,
+publish it from your jobs with the `test-results` CLI tool.
+
+Add the following snippet to your pipeline YAML file:
+
+``` yaml
+global_job_config:
+  epilogue:
+    always:
+      commands:
+         - [[ -f report.xml ]] && test-results publish report.xml
 ```
 
-This way if your test runner exits with non-zero status(due to the spec failure) report will be published.
+The `test-results` tool is part of [Semaphore's Toolbox]() and it is available
+in every Semaphore job.
+
+If you are running tests inside of a Docker container, see
+[how to use test-results CLI with Docker&nbsp;↗][working-with-docker]{target=_blank}
```

Comment:
This is probably the first time I see use of the arrow. Also we generally do target blank.  If you want to introduce it let's make some somehow official rules and propagate to other pages? Personally I think a normal link would be fine.

Reply:
The ↗ was inertia on my part to make a change. It was already in the document when I started editing.
I don't any strong preference, but now that you pointed it out, I do generally feel that they are just a distraction.

Thanks for pointing it out. Removing it 🔪 from this doc and from several other places.